lib/services/common-api-presenter.js

```diff
@@ -208,9 +213,10 @@ function CommonApiPresenterFactory(
                     } catch (error) {
                         // If we failed to render the error then we've got larger problems.
 
-                        logger.error('Unable to render error template.', {
-                            macaddress: options.macaddress,
-                            error: error
+                        logger.error('Unable to render error profile.', {
```

Comment:
The error occurs when both the input profile & boilerplate.ipxe rendering fail, but the error input profile is really something we care while analyzing log, so I don't think your log message is better than the original one.